src/DssSpell.sol

```diff
@@ -41,35 +51,176 @@ contract DssSpellAction is DssAction {
+        DssExecLib.setIlkMinAuctionBidIncrease("UNIV2DAIETH-A", 500);
+        DssExecLib.setIlkMinAuctionBidIncrease("UNIV2UNIETH-A", 500);
+        DssExecLib.setIlkMinAuctionBidIncrease("UNIV2ETHUSDT-A", 500);
+        DssExecLib.setIlkMinAuctionBidIncrease("UNIV2LINKETH-A", 500);
+        DssExecLib.setIlkMinAuctionBidIncrease("UNIV2WBTCDAI-A", 500);
+        DssExecLib.setIlkMinAuctionBidIncrease("UNIV2AAVEETH-A", 500);
+        DssExecLib.setIlkMinAuctionBidIncrease("UNIV2DAIUSDT-A", 500);
+
+        // RWA001-A collateral deploy
+        bytes32 ilk = "RWA001-A";
+
+        address vat = DssExecLib.vat();
+
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA001_A).vat() == vat, "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA001_A).ilk() == ilk, "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA001_A).gem() == RWA001_GEM, "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA001_A).dec() == DSTokenAbstract(RWA001_GEM).decimals(), "join-dec-not-match");
+
+        // init the RwaLiquidationOracle
+        // doc: "doc"
+        // tau: 5 minutes
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            ilk, RWA001_A_INITIAL_PRICE, DOC, 7890000
+        );
+        (,address pip,,) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+
+        // Set price feed for RWA001
+        DssExecLib.setContract(DssExecLib.spotter(), ilk, "pip", pip);
+
+        // Init RWA-001 in Vat
+        Initializable(vat).init(ilk);
+        // Init RWA-001 in Jug
+        Initializable(DssExecLib.jug()).init(ilk);
+
+        // Allow RWA-001 Join to modify Vat registry
+        DssExecLib.authorize(vat, MCD_JOIN_RWA001_A);
+
+        // Allow RwaLiquidationOracle to modify Vat registry
+        DssExecLib.authorize(vat, MIP21_LIQUIDATION_ORACLE);
+
+        // Increase the global debt ceiling by the ilk ceiling
+        DssExecLib.increaseGlobalDebtCeiling(1_000);
+        // Set the ilk debt ceiling
+        DssExecLib.setIlkDebtCeiling(ilk, 1_000);
+
+        // No dust
+        // DssExecLib.setIlkMinVaultAmount(ilk, 0);
+
+        // 3% stability fee
+        DssExecLib.setIlkStabilityFee(ilk, THREE_PCT_RATE, true);
+
+        // collateralization ratio 100%
+        DssExecLib.setIlkLiquidationRatio(ilk, 10_000);
+
+        // poke the spotter to pull in a price
+        DssExecLib.updateCollateralPrice(ilk);
+
+        // give the urn permissions on the join adapter
+        DssExecLib.authorize(MCD_JOIN_RWA001_A, RWA001_A_URN);
+
+        // set up the urn
+        Hopeable(RWA001_A_URN).hope(RWA001_OPERATOR);
 
-        // Increase PSM-USDC-A to 1 Billion from its current value.
-        DssExecLib.setIlkDebtCeiling("PSM-USDC-A", 1000 * MILLION);
+        // set up output conduit
+        Hopeable(RWA001_A_OUTPUT_CONDUIT).hope(RWA001_OPERATOR);
+        // could potentially kiss some BD addresses if they are available
```

Comment:
In the kovan spell we kiss some trust addresses on the output conduit before denying the deployer. We haven't done that on mainnet. We should check to see if there are any addresses we can kiss on the conduit in this spell.

Reply:
Since this first deploy is to test the circuit, we should make the first broker-dealer (trust) address the deployer address.  Once we add a real DC, we can `diss()` the deployer address and `kiss()` the broker-dealer address.  We can then test the circuit with that address for a small amount of DAI.

So, I suggest we `kiss()` the deployer address.

Reply:
Added.